odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -189,6 +189,28 @@ def test_merge_partner_archived(self):
         self.assertTrue(p2.exists())
         self.assertFalse(p3.exists())
 
+    def test_partner_merge_wizard_more_than_one_user_error(self):
+        """ Test that partners cannot be merged if linked to more than one user even if only one is active. """
+        Users = self.env['res.users']
+        Partner = self.env['res.partner']
+        MergeWizard = self.env['base.partner.merge.automatic.wizard']
+        u1, u2 = Users.create([{'name': 'test1', 'login': 'test1', 'partner_id': Partner.create({'name': 'test1'}).id},
+                               {'name': 'test2', 'login': 'test2', 'partner_id': Partner.create({'name': 'test2'}).id}])
+        dst_partner = Partner.create({'name': 'test3'})
+        partners = u1.partner_id + u2.partner_id + dst_partner
```

Comment:
can you create the 3 partners in batch directly ? (and then use `partners[0]` when you create the users)